Jenkinsfile

```diff
@@ -61,14 +86,12 @@ pipeline {
                                     "REGISTRY_PROJECT=${params.REGISTRY_PROJECT}",
                                     "REGISTRY=${params.REGISTRY_URI}")
 
-                            /* If the project's secret exists it must be removed at least in
-                             * the case of using a database along with an application.
-                             * Issue: Since the template is being process and applied with each
-                             * job run the secret is also being updated since it contains generate
-                             * values from the template.
```

Comment:
I'm not sure if I'd remove this part of the comment... it explains the logic around the removal of the secret...

I believe that the comment resulting from the change might not be clear enough... but it might be just me?